claim/models.py

```diff
@@ -1,17 +1,57 @@
 import datetime
 import json
 
+from django.db.models import Q
 from django.db import models
 from django.contrib.auth.models import User
-from django.conf import settings
 from django.db.models import Count
 from django.utils.translation import ugettext as _
 
 
+class Organization(models.Model):
+    id = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=255)
+
+    responsible_email = models.CharField(max_length=255)
+    responsible_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+    def get_json_claims(self):
+        # Q(polygon_id=self.id) jsut compatible layer.
+        # Remove me after release. Not critical.
+        claims = Claim.objects.filter(Q(organization=self) |
+                                      Q(polygon_id=self.id))
+        claims_list = []
+
+        if claims:
+            for claim in claims:
+                username = claim.complainer.username if\
+                    claim.complainer else _("Anon")
+                claims_list.append({
+                    'text': claim.text,
+                    'servant': claim.servant,
+                    'complainer': username
+                })
+
+        return json.dumps(claims_list)
+
+    @classmethod
+    def get_n_create(cls, polygon_id):
+        try:
+            org = cls.objects.get(id=polygon_id)
+        except cls.DoesNotExist:
```

Comment:
Maybe better create some sync function, that will update organization table after update of geojson?